packages/okta-angular/src/okta/okta.service.ts

```diff
@@ -147,21 +152,21 @@ export class OktaAuthService {
     /**
      * Clears the user session in Okta and removes
      * tokens stored in the tokenManager.
+     * @param uri 
      */
-    async logout() {
+    async logout(uri?: string) {
       this.oktaAuth.tokenManager.clear();
       await this.oktaAuth.signOut();
+      this.router.navigate([uri || '/']);
     }
 
     /**
      * Scrub scopes to ensure 'openid' is included
+     * @param scopes
      */
-    scrubScopes(scopes) {
+    scrubScopes(scopes: string) {
       if (!scopes) {
         return 'openid email';
-      } else {
-        // Make sure object is a string
-        scopes = Array.isArray(scopes) ? scopes.join(' ') : scopes
```

Comment:
Before merging, let's continue our conversation about conventional commits.  I think this breaking change needs to be it's own commit, and the change to loginRedirect() needs to be it's own commit.  There's a lot going on in this PR and someone scanning for breaking changes may have a hard time finding them.